scripts/slack/notify_deploy.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+import sys
+import requests
+
+def notifySlack(image_name, deployment_env_name):
+    circle_build_url = os.getenv('CIRCLE_BUILD_URL')
+    url = os.getenv('SLACK_WEBHOOK')
+
+    data = {
+      'attachments': 
+      [
+        {
+          'color': '#7CD197',
+          'fallback': 'Build Notification: ' + circle_build_url,
+          'title': 'Kubernetes-Monitor Deploy Notification',
+          'text': ':hatching_chick: Deploying Kubernetes-Monitor on `'+ deployment_env_name + '`:`' + image_name + '`: :hatching_chick:'
```

Comment:
I think it needs a space after the colon (if you look at the shell script it had a space):
```python
'`: `'
```
There also isn't a colon before the chick emoji:
```python
` :hatching_chick:'
```

Reply:
The space after the colon is missing though!
Currently:
```python
deployment_env_name + '`:`' + image_name
```
How it should be:
```python
deployment_env_name + '`: `' + image_name
```

Reply:
yes, sorry, I've added it in the wrong place 🤦‍♀